spec/features/questions_feature_spec.rb

```diff
@@ -53,6 +54,13 @@
     expect(page).to have_content "1 error prohibited this question from being saved:"
   end
 
+  it 'disallows non owners to make event questions' do
+    click_on "Sign out"
+    sign_in_as(user_two)
+    visit event_path(event)
```

Comment:
This should probably be:  
`visit new_event_question_path(event)`  
Direct targets the proper path. The current line tests that the user can't visit someone else's event.